populate_students.py

```diff
@@ -1,25 +1,25 @@
 import random
```

Comment:
Is this part of the task? Or This is another issue and should solve in another branch